bukkit/src/main/java/com/github/games647/fastlogin/bukkit/listener/protocollib/VerifyResponseTask.java

```diff
@@ -171,17 +175,33 @@ private Object getNetworkManager() throws IllegalAccessException, ClassNotFoundE
     }
 
     private boolean enableEncryption(SecretKey loginKey) throws IllegalArgumentException {
+        if (encryptMethod == null) {
+            Class<?> networkManagerClass = MinecraftReflection.getNetworkManagerClass();
+
+            try {
+                encryptMethod = FuzzyReflection.fromClass(networkManagerClass)
+                        .getMethodByParameters("a", SecretKey.class);
+            } catch (IllegalArgumentException exception) {
+                encryptMethod = FuzzyReflection.fromClass(networkManagerClass)
+                        .getMethodByParameters("a", Cipher.class, Cipher.class);
+
+                Class<?> encryptionClass = MinecraftReflection.getMinecraftClass("MinecraftEncryption");
+                cipherMethod = FuzzyReflection.fromClass(encryptionClass)
```

Comment:
Please put in a comment what those two methods actually do. So the first one enables encryption, while the other initializes the cipher.

Reply:
I usually don't comment the code much - comments added now

Reply:
In this context it was really clear what it does, so it was really necessary.